Week4/app/src/main/java/com/kulloveth/moviesapp/db/room/MovieDatabse.kt

```diff
@@ -46,9 +47,8 @@ abstract class MovieDatabse : RoomDatabase() {
         ).addCallback(object : RoomDatabase.Callback() {
             override fun onCreate(db: SupportSQLiteDatabase) {
                 super.onCreate(db)
-                GlobalScope.launch(context = Dispatchers.IO) {
```

Comment:
The assignment said not to use GlobalScope, is there another way to do this?

Reply:
Oh this skipped my mind of Course there are lot of ways to do that, Thanks

Reply:
` private fun builDatabase() = Room.databaseBuilder(
            context,
            MovieDatabse::class.java,
            "movie_database"
        ).addCallback(object : RoomDatabase.Callback() {
            override fun onCreate(db: SupportSQLiteDatabase) {
                super.onCreate(db)
                CoroutineScope(context = Dispatchers.IO).launch {
                    Injection.provideRepository.insertAllMovie(MoviesDataManager.movieList)
                }
            }
        }).build()
    }`
This is what I have in my local,I guess I forgot to update that